x/registration/remoteAttestation/remote_attestation.go

```diff
@@ -184,14 +181,7 @@ func verifyAttReport(attnReportRaw []byte, pubK []byte) ([]byte, error) {
 				if err != nil && len(platInfo) != 105 {
 					return nil, errors.New("illegal PlatformInfoBlob")
 				}
-				platInfo = platInfo[4:]
-
-				//piBlob := parsePlatform(platInfo)
```

Comment:
I think we want this for debugging porpuses?

https://github.com/scrtlabs/SecretNetwork/pull/954/files#r928222109

Reply:
ahhh ok :)